api/caaCaseTypes/caaCaseTypes.util.ts

```diff
@@ -1,5 +1,16 @@
 import { CaaCasesPageType } from '../caaCases/enums';
-import { searchCasesString } from './caaCaseTypes.constants';
+import { searchCasesString, validCaseTypeIdRegex, dummyCaseTypeId } from './caaCaseTypes.constants';
+import * as log4jui from '../lib/log4jui';
+
+const logger = log4jui.getLogger('user-types');
+
+export function sanitizeCaseTypeId(caseTypeId: string): string {
+  const out = caseTypeId === null ? null : (caseTypeId.match(validCaseTypeIdRegex) === null ? dummyCaseTypeId : caseTypeId);
+  if (out !== caseTypeId) {
+    logger.warn('Case Type Id with unacceptible characters detected. Will be sanitized with default dummy value replacement.');
```

Comment:
```suggestion
    logger.warn(`Case Type Id '${caseTypeId}' with unacceptable characters detected. Will be sanitized with default dummy value replacement.`);
```

Reply:
Logging a bad ID input would necessitate the logged value to be masked into sanitized value as otherwise static scans would complain about them too. Therefore I skipped logging the bad ID.

Reply:
Ah yeah, of course 😄